### PR TITLE
add vacation functionality

### PR DESCRIPTION
**Pull request by MikeRoetgers**
Define vacations so developers who are absent don't get PRs assigned.

### CODE REVIEW

**Comment by husio** [comment]
IMO vacation days should be read from external file, we could put there what's already planned.

**Comment by MikeRoetgers** [comment]
I'm kinda doing that. A crontab entry on util1 looks like this:

```
docker rm github-stale-pr-bot; docker run --name "github-stale-pr-bot" docker-registry.optiopay.com/github-stale-pr-bot -auth-key `cat ~/.githubbot-auth-key` -stale 1m -old 18h -vacation `cat ~/.githubbot-vacations`
```

But yeah, I can also change that completely and provide a filepath instead of the data itself.

File: main.go [hunk 1]
```diff
@@ -132,6 +134,17 @@ func stalePullRequests(staleTime time.Duration) (stale []Issue, err error) {
 	return stale, nil
 }
 
+type VacationUsers []string
```
Comment (satran) [inline main.go:137]: You could have used a `map[string]bool` for this and avoided the `Contains` method.

**Comment by husio** [comment]
@MikeRoetgers then maybe we should switch to YAML configuration file, now that the amount of flags is rather big?

**Comment by MikeRoetgers** [comment]
Ok, let's put this in hold. I will switch to a config file for the service.

**Pull request closed without merging**